crates/driver/src/tests/setup/blockchain.rs

```diff
@@ -688,55 +684,66 @@ async fn primary_account(web3: &DynWeb3) -> ethcontract::Account {
     ethcontract::Account::Local(web3.eth().accounts().await.unwrap()[0], None)
 }
 
-/// An instance of geth managed by the `dev-geth` crate. When this type is
-/// dropped, the geth instance gets shut down.
-#[derive(Debug)]
-pub struct Geth {
-    port: String,
+/// A blockchain node for development purposes. Dropping this type will
+/// terminate the node.
+pub struct Node {
+    #[allow(dead_code)] // store `process` to later run its `Drop` handler
+    process: tokio::process::Child,
```

Comment:
I _think_ you can also suppress the warning this way:

```suggestion
    _process: tokio::process::Child,
```

I'm indifferent on which one to use - just point it out.

Reply:
Yeah, just felt that this way documents it a bit nicer. Also you wouldn't have to change the identifier should you ever have to use that variable (i.e. smaller diff).